app.js

```diff
@@ -226,43 +224,55 @@ const runDaemon = async () => {
     });
 };
 
-// CLI definition
-const y = yargs(hideBin(process.argv));
-y.scriptName("elprimobot");
-y.option("verbose", {
-    alias: "v",
-    type: "boolean",
-    description: "Run with verbose logging",
-});
-y.command({
-    command: "leetcode-daily",
-    description: "Sends leetcode daily problem to a Discord channel",
-    handler: async () => {
-        await postDailyMessages();
-    },
-});
-y.command({
-    command: "stats-daily",
-    description: "Sends Discord users daily chat activity stats",
-    handler: async () => {
-        const stats = new Stats(discordClient);
-        await stats.postDailyStats();
-    },
-});
-y.command({
-    command: "stats-weekly",
-    description: "Sends Discord users weekly chat activity stats",
-    handler: async () => {
-        const stats = new Stats(discordClient);
-        await stats.postWeeklyStats();
-    },
-});
-y.command({
-    command: "daemon",
-    description: "Runs the all the features in blocking mode",
-    handler: async () => {
-        await runDaemon();
-    },
-});
+async function main() {
+    const discordClient = await getDiscordClient();
+    // CLI definition
+    const y = yargs(hideBin(process.argv));
+    y.scriptName("elprimobot");
+    y.option("verbose", {
+        alias: "v",
+        type: "boolean",
+        description: "Run with verbose logging",
+    });
+    y.command({
+        command: "leetcode-daily",
+        description: "Sends leetcode daily problem to a Discord channel",
+        handler: async () => {
+            await postDailyMessages();
```

Comment:
should we call `leetcode post` instead of postDaily which also includes the stats?

Reply:
Maybe, let's have a look later in the pairing